packages/driver/src/cy/commands/files.ts

```diff
@@ -110,7 +110,14 @@ export default (Commands, Cypress, cy, state) => {
           }
 
           // Add the filename as a symbol, in case we need it later (such as when storing an alias)
-          state('current').set('fileName', basename(filePath))
+          try {
```

Comment:
Should we take this chance to stop importing `path`  in the driver, and just copy-paste a minimal `basename` function? I don't think driver should be coupled to a polyfill, or webpack for that matter. We bundle a huge polyfill for a tiny amount of behavior.

In fact, we should be able to eliminate webpack entirely from our build process pretty soon. Only driver and reporter use it, but those could go to Vite pretty soon.

Reply:
We will still need a buffer implementation, but we could explicitly import that, instead of relying on the bundler to do it for us.

Reply:
I agree that we should limit down the use of polyfills we don't really need, especially ones that large. Probably in this case we could just import the tree shakable version from the polyfill and exclude it from webpack. However, since we already have a lot going on here, I'd prefer not to touch it in this PR and keep things as 1:1 as possible. After we get something in, we can create a follow up to strip down some of this, which makes the changeset a bit smaller and clearer.

Reply:
Sounds good. I don't know how much of `path` we use but if we only need one function we can just grab the code for it, that would be fine too - let's revisit this after.